tensorflow_encrypted/protocol/pond.py

```diff
@@ -914,6 +914,11 @@ def unwrapped(self):
     def reveal(self):
         return self.prot.reveal(self)
 
+    @staticmethod
+    def zero(prot: Protocol, shape: List[int]) -> 'PondPrivateTensor':
```

Comment:
Unsure about doing this this way.  I wanted a zero method as a PondVariable though as I don't think we should be forced to use Int100Tensors with PondTensors.

Reply:
I think related to the work I'm doing now there will be a static method on the `AbstractTensor` that will force subclasses of `AbstractTensor` to implement the zeros function for the given tensor.